man/paru.conf.5

```diff
@@ -145,27 +146,31 @@ to use https://aur.tuna.tsinghua.edu.cn/.
 
 .TP
 .B BuildDir = /path/to/dir
-Directory to use for Building AUR Packages. This directory is also used as
+Directory to use for building AUR Packages. This directory is also used as
 the AUR cache when deciding if paru should skip builds.
 
 .TP
-.B SortBy = Votes|Popularity|Id|Baseid|Name|Base|Submitted|Modified
+.B SortBy = votes|popularity|name|base|submitted|modified|id|baseid
+Defaults to votes. Sort AUR results according to the options in "Sort by"
+visible here: https://aur.archlinux.org/packages/
 
 .TP
-.B SearchBy = name|name-desc|maintainer|depends|checkdepends|makedepends|optdepends>
+.B SearchBy = name|name-desc|maintainer|depends|checkdepends|makedepends|optdepends
+Defaults to name-desc. Search AUR packages according to the options in 
+"Search by" visible here: https://aur.archlinux.org/packages/    
 
 .TP
-.B CompletionInterval = N
+.B CompletionInterval = Number
 Time in days to refresh the completion cache. Setting this to 0 will cause
 the cache to be refreshed every time, while setting this to -1 will cause the
-cache to never be refreshed.
+cache to never be refreshed. Defaults to 7.
 
 .TP
 .B RequestSplit = Number
```

Comment:
Again, this option was removed. Looks like the option is still there in the code, but that will be removed.